src/Lykke.Job.TransactionHandler/Projections/OperationHistoryProjection.cs

```diff
@@ -60,7 +60,7 @@ public class OperationHistoryProjection
 
         public async Task Handle(TransferOperationStateSavedEvent evt)
         {
-            await _log.WriteInfoAsync(nameof(OperationHistoryProjection), nameof(TransferOperationStateSavedEvent), evt.ToJson(), "");
+            _log.WriteInfo(nameof(TransferOperationStateSavedEvent), evt.ToJson(), "");
```

Comment:
This is not needed with latest version of Lykke.Cqrs - such logging is done from the library